plugins/modules/k8s_log.py

```diff
@@ -60,6 +60,11 @@
     required: no
     type: str
     version_added: '2.2.0'
+  previous:
+    description:
+    - If true, print the logs for the previous instance of the container in a pod if it exists.
+    required: no
+    type: bool
```

Comment:
Just realized this also needs a `version_added: '2.4.0'`.

Reply:
This has been added.